### PR TITLE
Suport to css modules in product-images

### PR DESCRIPTION
**Pull request by GustavoMA**
#### What is the purpose of this pull request?

As described in the title.

#### How should this be manually tested?
[https://cssmodulesproductimages--storecomponents.myvtex.com](https://cssmodulesproductimages--storecomponents.myvtex.com)
#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductImages/components/Carousel/index.js [hunk 1]
```diff
@@ -206,12 +208,12 @@ class Carousel extends Component {
       resistanceRatio: slides.length > 1 ? 0.85 : 0,
       renderNextButton: () => (
         <span className={`swiper-caret-next pl7 right-1 ${caretClassName}`}>
-          <Icon id="nav-angle--right" size={24} className="vtex__icon-caret-right" />
+          <Icon id="nav-angle--right" size={24} className={styles.carouselIconCaretRight} />
```
Comment (JuanAlmeida12) [inline react/components/ProductImages/components/Carousel/index.js]: I think is better to remove this magic number `24`.

File: react/components/ProductImages/components/Carousel/styles.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+.carouselGaleryCursor {
+  cursor: url('./cursor.svg') 8 8, default;
+}
+
+.carouselGaleryThumbs :global(.swiper-slide) {
+  height: auto;
+}
+
+.carouselGaleryThumbs :global(.swiper-slide):not(:global(.swiper-slide-thumb-active)) .carouselThumbBorder {
+  border-width: 0px;
```
Comment (JuanAlmeida12) [inline react/components/ProductImages/components/Carousel/styles.css]: It's possible to use tachyons for this?

File: react/components/ProductImages/components/Carousel/styles.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+.carouselIconCaretRight,
+.carouselIconCaretLeft {
+  stroke: white;
+  stroke-width: 0.5px;
+  -webkit-filter: drop-shadow( 0px 1px 1px #CACBCC );
+  filter: drop-shadow( 0px 1px 1px #CACBCC );
+}
+
+.carouselCursorDefault:hover{
+  cursor: default;
```
Comment (JuanAlmeida12) [inline react/components/ProductImages/components/Carousel/styles.css]: Here too

Comment (GustavoMA) [reply react/components/ProductImages/components/Carousel/styles.css]: I did't find any cursor related Tachyons :crying_cat_face:

File: react/components/ProductImages/components/Carousel/styles.css (hunk 2 above)
Comment (GustavoMA) [reply react/components/ProductImages/components/Carousel/styles.css]: This one is changing from an external lib, so...

File: react/components/ProductImages/components/Carousel/index.js (hunk 1 above)
Comment (GustavoMA) [reply react/components/ProductImages/components/Carousel/index.js]: Good one. Fixed.

**Pull request merged**